frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -141,7 +141,7 @@ export default function CohortsPage() {
           sampleQueryParamValue &&
           `${sampleQueryParamHeaderName} "${sampleQueryParamValue}"`
         }
-        getSamplesRowData={getCohortDataFromSamples}
+        getSamplesRowData={getSampleCohortDataFromSamplesQuery}
```

Comment:
This is a function that takes data returned from the GraphQL query that fetches samples data, and then transforms that data to fit into the AG Grid table. Renamed for improved clarity.

(Similar comment for the Patients, Requests, and Samples page files directly below.)